src/code/z_fcurve_data_skelanime.c

```diff
@@ -1,16 +1,188 @@
 #include <ultra64.h>
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fcurve_data_skelanime/func_8006C750.s")
+void SkelCurve_Zero(SkelAnimeCurve* skelCurve) {
```

Comment:
I think a name like Clear or Reset would be more appropriate.
```suggestion
void SkelCurve_Clear(SkelAnimeCurve* skelCurve) {
```